### PR TITLE
Update Neox Scanlator & Added Neoxxx Scanlator

### LINKED ISSUE #3753: [Neoxscans] Connector not working
**Did you read [the troubleshooting guide](https://hakuneko.download/docs/troubleshoot/)**
Yes

**Is the website of the connector working properly / are you able to see the manga within your browser**
Yes

**Describe the bug**
Site change neoxscans.com for neoxscans.net

is it possible to leave the 2 domains?? I noticed that the site is floating to .com and .net.

**HakuNeko (please complete the following information):**
 - Version [e.g. 6.17]

### PR DESCRIPTION
**Pull request by Slowftw**
Fix #3753

### CODE REVIEW

**Comment by Slowftw** [comment]
Resolves #3754

File: src/web/mjs/connectors/NeoxScan.mjs [hunk 1]
```diff
@@ -8,5 +8,10 @@ export default class NeoxScan extends WordPressMadara {
         super.label = 'Neox Scanlator';
         this.tags = [ 'manga', 'webtoon', 'portuguese', 'scanlation' ];
         this.url = 'https://neoxscans.com';
+        this.queryTitleForURI = '.post-title';
     }
-}
+
+    canHandleURI(uri) {
+        return /https?:\/\/(xxx\.)?neoxscans\.(com|net)/.test(uri.origin);
```
Comment (27136773) [inline src/web/mjs/connectors/NeoxScan.mjs]: Maybe add https://xxx.neoxscans.net as a new website?

File: src/web/mjs/connectors/NeoxxxScan.mjs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class NeoxScan extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'neoxxxscan';
+        super.label = 'Neoxxx Scanlator';
+        this.tags = [ 'manga', 'webtoon', 'portuguese', 'scanlation' ];
```
Comment (27136773) [inline src/web/mjs/connectors/NeoxxxScan.mjs]: May add `hentai` tag

**Review by 27136773 — APPROVED** [review]
Looks good, minor improvments possible

**Pull request merged**